apps/meteor/client/components/Omnichannel/modals/CloseChatModal.tsx

```diff
@@ -206,13 +206,16 @@ const CloseChatModal = ({
 								)}
 							</>
 						)}
+						<Field marginBlockStart='x16'>
+							<Field.Label color='annotation' fontScale='c1'>
+								{canSendTranscriptPDF && hasLicense && canSendTranscriptEmail && visitorEmail
```

Comment:
Throwing all this variables into a single one might help with the code readability